repository/src/main/java/tech/ydb/yoj/repository/db/AbstractDelegatingTable.java

```diff
@@ -26,6 +26,10 @@ protected AbstractDelegatingTable() {
         this.target = BaseDb.current(BaseDb.class).table(resolveEntityType());
     }
 
+    protected AbstractDelegatingTable(String tableName) {
```

Comment:
- Please mark this constructor as `@ExperimentalApi(issue="https://github.com/ydb-platform/yoj-project/issues/32")`.
- Please explicitly annotate `tableName` with Lombok's `@NonNull` (I don't think it makes sense to call this constructor with `tableName == null`, even if the `BaseDb.table(entity class, table name)` method allows it.)